lib/krane/restart_task.rb

```diff
@@ -167,6 +167,12 @@ def identify_target_deployments(selector: nil)
         apps_v1_kubeclient.get_deployments(namespace: @namespace, label_selector: selector_string)
       end
       deployments.select { |d| d.dig(:metadata, :annotations, ANNOTATION) }
+      .map do |d|
+        d["apiVersion"] = "apps/v1"
+        d["kind"] = "Deployment"
```

Comment:
why do we need to set/override these now?

Reply:
The response doesn't contain these values and we now depend on these values being set.

Reply:
Ah right, we got them from Kubeclient instead of kubectl here. Related issue: https://github.com/ManageIQ/kubeclient/issues/368